dist/index.js

```diff
@@ -29231,18 +29231,14 @@ function createBlueprint(themeSlug, branch, repo, themeDir) {
     /* If themeDir is not provided, we assume that the action is running in a single theme workflow and the theme folder name will be the theme slug + the branch name.
      * If themeDir is provided, we assume that the action is running in a multi theme workflow and the theme folder name will be the theme slug.
      */
-    const sanitizedBranch = branch.replace(/[\/\\:*?"<>|]/g, '-');
-    const themeFolderName = !themeDir
-        ? `${themeSlug}-${sanitizedBranch}`
-        : themeSlug;
-    (0, core_1.debug)(`Theme folder name: ${themeFolderName}`);
     const wpVersion = (0, core_1.getInput)('wp-version');
     const phpVersion = (0, core_1.getInput)('php-version');
     const preferredVersions = {
         ...(wpVersion && { wp: wpVersion }),
         ...(phpVersion && { php: Number(phpVersion).toFixed(1) }),
     };
     const template = {
+        ...(Object.keys(preferredVersions).length && { preferredVersions }),
         preferredVersions,
```

Comment:
I'm confused here? Why would we need both?

Would you mind adding a comment explaining this?